app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoMegaTile.kt

```diff
@@ -23,11 +25,21 @@ class PocketVideoMegaTile(
 ) : LinearLayout(context, attrs) {
 
     var pocketVideos by Delegates.observable<List<PocketVideo>?>(null) { _, _, newVideos ->
-        if (newVideos == null) return@observable
-        thumbnailViews.forEachIndexed { i, thumbnailView ->
-            PicassoWrapper.client.load(newVideos[i].thumbnailURL)
-                    .transform(roundCornerTransformation)
-                    .into(thumbnailView)
+        // When no Pocket API key is provided, show placeholder tiles (developer ergonomics)
+        if (BuildConfig.POCKET_KEY == null) {
+            thumbnailViews.forEachIndexed { _, thumbnailView ->
+                PicassoWrapper.client.load("https://blog.mozilla.org/firefox/files/2017/12/Screen-Shot-2017-12-18-at-2.39.25-PM.png")
+                        .transform(roundCornerTransformation)
+                        .into(thumbnailView)
+            }
+            Toast.makeText(context, "Pocket API key was not found.", Toast.LENGTH_LONG).show()
+        } else {
+            if (newVideos == null) return@observable
+            thumbnailViews.forEachIndexed { i, thumbnailView ->
+                PicassoWrapper.client.load(newVideos[i].thumbnailURL)
+                        .transform(roundCornerTransformation)
+                        .into(thumbnailView)
+            }
```

Comment:
Optional nit: I think the readability is more or less a wash between these two, but we could reduce some of the repetition by doing something like the following.  Up to you which you prefer.

```kotlin
    var pocketVideos by Delegates.observable<List<PocketVideo>?>(null) { _, _, newVideos ->
        val thumbnails = if (BuildConfig.POCKET_KEY == null) {
            Toast.makeText(context, "Pocket API key was not found.", Toast.LENGTH_LONG).show()
            (0 until thumbnailViews.size)
                    .map { "https://blog.mozilla.org/firefox/files/2017/12/Screen-Shot-2017-12-18-at-2.39.25-PM.png" }
        } else {
            newVideos?.map { it.thumbnailURL } ?: return@observable
        }
        thumbnailViews.forEachIndexed { i, thumbnailView ->
            PicassoWrapper.client.load(thumbnails[i])
                    .transform(roundCornerTransformation)
                    .into(thumbnailView)
        }
    }
```